modules/common/src/environments/environments.ts

```diff
@@ -0,0 +1,42 @@
+import {EnvironmentType} from './interfaces';
+
+export class Environments {
+    public static getEnv (): EnvironmentType {
+        return process.env.NODE_ENV === EnvironmentType.PROD ?
+            EnvironmentType.PROD :
+            process.env.NODE_ENV === EnvironmentType.TEST ?
+                EnvironmentType.TEST :
+                process.env.NODE_ENV === EnvironmentType.BETA ?
+                    EnvironmentType.BETA :
+                    EnvironmentType.DEV;
+    }
+
+    public static isProd (): boolean {
+        return this.getEnv() === EnvironmentType.PROD || this.getEnv() === EnvironmentType.BETA;
+    }
+
+    public static isDev (): boolean {
+        return this.getEnv() === EnvironmentType.DEV;
+    }
+
+    public static isTest (): boolean {
+        return this.getEnv() === EnvironmentType.TEST;
+    }
+
+    public static getBugsnagKey (): string {
+        return 'f85a2d2409fa127df8ac0d51b4c81f0d';
```

Comment:
what about this key?

Reply:
Oof.. I will move it to `env-consts` within the service actually